spec.html

```diff
@@ -33814,29 +33819,30 @@ <h1>String.prototype.padStart ( _maxLength_ [ , _fillString_ ] )</h1>
         <p>When the `padStart` method is called, the following steps are taken:</p>
         <emu-alg>
           1. Let _O_ be ? RequireObjectCoercible(*this* value).
-          1. Return ? StringPad(_O_, _maxLength_, _fillString_, ~start~).
+          1. Let _S_ be ? ToString(_O_).
+          1. Let _maxLen_ be ℝ(? ToLength(_maxLength_)).
+          1. If _fillString_ is *undefined*, let _filler_ be ~empty~.
+          1. Else, let _filler_ be ? ToString(_fillString_).
+          1. If _filler_ is ~empty~, let _filler_ be the String value consisting solely of the code unit 0x0020 (SPACE).
```

Comment:
No need for this to be a separate step; you can just do `If _fillString_ is *undefined*, let _filler_ be the String value consisting solely of the code unit 0x0020 (SPACE)` above.

Reply:
oh right, duh, thanks